security/backends/elasticsearch/is_core/filters.py

```diff
@@ -22,6 +22,15 @@ class UsernameUserFilter(ElasticsearchFilter):
 
     def clean_value(self, value, operator_slug, request):
         user_model = get_user_model()
+        queryset = user_model.objects.filter(
+            **{'{}__contains'.format(user_model.USERNAME_FIELD): value}
+        ).annotate(
+            str_id=Cast('id', output_field=TextField())
+        ).values_list('str_id', flat=True)
+
+        if len(queryset) > 65536:
```

Comment:
Why 65536? It should be a constant variable with the correct name.

Reply:
this is constant value in elastic, but yes I can add this to the settings